knowledge_base/job_read_secret/databricks.yml

```diff
@@ -0,0 +1,47 @@
+bundle:
+  name: job-read-secret-example
+
+# workspace:
+#   host: https://myworkspace.cloud.databricks.com
+
+resources:
+  secret_scopes:
+    my_secret_scope:
+      name: secrets-scope-1
+      permissions:
+        - level: CAN_VIEW
+          group_name: users
+        - level: CAN_MANAGE
```

Comment:
Admins should have these permissions by default. Is there a reason to explicitly specify them?

Reply:
Is this only limited to read permissions:
https://arc.net/l/quote/fotmmsku

Reply:
I'm referring to permissions that are listed by secret scopes ACL - the only initial permission a new secret scope is getting, is that its create can MANAGE it. For other permissions to show up there, they need to be explicitly declared in the `permissions` section

Reply:
Okay, but do admins have MANAGE permission by default on the scope anyways? The docs seem to imply so.

Note: This is not blocking, an additional permissions block can't really hurt customers. I was curious about the intention though.

Reply:
the API doc does not say so: https://docs.databricks.com/api/workspace/secrets/createscope

only the initial_manage_principal is getting the MANAGE permission, and when created with DABS, it is always the caller